source/MRViewer/MRSurfaceManipulationWidget.cpp

```diff
@@ -280,7 +276,7 @@ void SurfaceManipulationWidget::updateRegion_( const Vector2f& mousePos )
 
     const auto& viewerRef = getViewerInstance();
     std::vector<Vector2f> viewportPoints;
-    if ( !mousePressed_ || ( mousePos - mousePos_ ).lengthSq() < 9.f )
+    if ( !mousePressed_ || ( mousePos - mousePos_ ).lengthSq() < 16.f )
```

Comment:
16.0f + // 4px radius

Reply:
yes. I thought it would be better to slightly increase threshold